tests/unit/validation.test.ts

```diff
@@ -0,0 +1,88 @@
+import * as Joi from 'joi';
+import { messagevalidation as schema } from '../../src/chatbot/validation';
+
+const defaultMessage = {
+  message: 'x',
+  meta: {
+    x: '1',
+  },
+  send: true,
+  session_id: '11111',
+  type: 'mytype',
+};
+
+const hasError = (obj, key, force = false) => {
```

Comment:
Het was mij niet meteen duidelijk wat die `force` flag doet. Eventueel hernoemen naar `mergeDefaults = true` of iets dergelijks als je dat ook duidelijker vindt.